.travis.yml

```diff
@@ -145,7 +145,8 @@ script:
     fi
 
   - # run all of the tests, making sure failures at this stage don't cause travis failures
-  - python devtools/run-ctest.py -j2 --schedule-random --stop-time $CTEST_STOP_TIME
+  - # set a timeout of $CTEST_TIMEOUT seconds for each individual test
+  - python devtools/run-ctest.py -j2 --schedule-random --stop-time $CTEST_STOP_TIME --timeout $CTEST_TIMEOUT
   - if [[ ! -z "${DOCS_DEPLOY}" && "${DOCS_DEPLOY}" = "true" ]]; then
```

Comment:
It doesn't look like like the environment variable CTEST_TIMEOUT was ever set though?